src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -166,7 +166,6 @@
     "Microsoft.DiaSymReader.Native.x86.dll",
     "mscordaccore.dll",
     "mscordbi.dll",
-    "mscorrc.dll",
```

Comment:
This list includes all files that ever shipped in the platform, even the ones that are no longer shipping. I am not sure whether it is actually needed, but we may want to keep it for consistency in this list.